Android/KotlinxCoroutines/source/Kotlinx.Coroutines.Core.Jvm/Kotlinx.Coroutines.Core.Jvm.csproj

```diff
@@ -0,0 +1,55 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I don't think we want to bind these libraries that aren't intended for Android:
- Xamarin.Kotlinx.Coroutines.Core.Jvm
- Xamarin.Kotlinx.Coroutines.Core.Jdk8
- Xamarin.Kotlinx.Coroutines.Core.Reactive
- Xamarin.Kotlinx.Coroutines.Core.Rx2

Today it looks like we only publish 2 packages:
- Xamarin.KotlinX.Coroutines.Core
- Xamarin.KotlinX.Coroutines.CoroutinesAndroid

https://www.nuget.org/packages?q=xamarin+kotlin+coroutine

Reply:
> Today it looks like we only publish 2 packages:

I think I bound those from Kotlin folder, just minimal dependencies I needed for GPS-FB-MLKit.

So, deleting or not publishing other packages?

Reply:
Dependencies listed:

*   `kotlinx-coroutines-android`

    *   dependencies

        *   `kotlin-stdlib`

        *   `kotlinx-coroutines-core-jvm`        

*   `kotlinx-coroutines-core-jvm`

    *   dependencies

        *   `kotlin-stdlib`

        *   `kotlin-stdlib-common`

*   `kotlinx-coroutines-jdk8`

    *   dependencies

        *   `kotlin-stdlib`

        *   `kotlinx-coroutines-core-jvm`        

*   `kotlinx-coroutines-reactive`

    *   dependencies

        *   `kotlin-stdlib`

        *   `kotlinx-coroutines-core-jvm`

        *   `org.reactivestreams:reactive-streams`

*   `kotlinx-coroutines-rx2`

    *   dependencies

        *   `kotlin-stdlib`

        *   `kotlinx-coroutines-core-jvm`

        *   `io.reactivex.rxjava2:rxjava`        

        *   `kotlinx-coroutines-reactive`

Already bound:

*  `Kotlin.StdLib`
*  `Xamarin.Android.ReactiveStreams`
*  `RxJava`

So seem like those could be used on Android and it